client/blocks/comment-detail/index.jsx

```diff
@@ -82,16 +82,31 @@ export class CommentDetail extends Component {
 
 	toggleApprove = () => {
 		const { commentId, commentStatus, postId, setCommentStatus } = this.props;
-		setCommentStatus( commentId, postId, 'approved' === commentStatus ? 'unapproved' : 'approved' );
+		const isPersisted = 'approved' === commentStatus || 'unapproved' === commentStatus;
+
+		setCommentStatus( commentId, postId, 'approved' === commentStatus ? 'unapproved' : 'approved', {
```

Comment:
it would have been nice if we had included `siteId` and `postId` in the faker so that we wouldn't have had to make any changes here now. if we will need them anyway, why isn't `siteId` here?